docs/pages/transactions/solana/api-prepare.mdx

```diff
@@ -0,0 +1,47 @@
+<CodeBlocks>
+
+```ts twoslash title="example.ts"
+import * as solanaWeb3 from "@solana/web3.js";
+import { connection, keypair } from "./config.ts";
+
+const instructions = [
+  solanaWeb3.SystemProgram.transfer({
+    fromPubkey: keypair.publicKey,
+    toPubkey: keypair.publicKey,
+    lamports: 5,
+  }),
+];
+
+const { blockhash } = await connection.getLatestBlockhash();
+
+const message = new solanaWeb3.TransactionMessage({
+  payerKey: new solanaWeb3.PublicKey(
+    "Amh6quo1FcmL16Qmzdugzjq3Lv1zXzTW7ktswyLDzits",
+  ), // Placeholder: Set this to an address different than any other address included in the tx, will be replaced by alchemy's feePayer
```

Comment:
nit: i would put the comment above the value since it wrapped to another line
```suggestion
  payerKey: new solanaWeb3.PublicKey(
    // Placeholder: Set this to an address different than any other address included in the tx, will be replaced by alchemy's feePayer
    "Amh6quo1FcmL16Qmzdugzjq3Lv1zXzTW7ktswyLDzits",
  ),
```